lib/Service/SuperService.php

```diff
@@ -16,4 +16,39 @@ public function __construct(LoggerInterface $logger, ?string $userId, Permission
 		$this->logger = $logger;
 		$this->userId = $userId;
 	}
+
+	public function AdminNotificationCall($tableId,$viewId) {
```

Comment:
`AdminNotificationCall` is not descriptive.

Reply:
Let me change it.